src/lib/prisma-session-store.ts

```diff
@@ -377,12 +461,13 @@ export class PrismaSessionStore<M extends string = 'session'> extends Store {
       })
       .catch(() => null);
 
-    const data = {
+    let data: ICreatePrismaSession = {
       sid,
       expiresAt,
       data: sessionString,
       id: this.dbRecordIdIsSessionId ? sid : this.dbRecordIdFunction(sid),
     };
+    if (uid) data = { ...data, uid };
```

Comment:
Typically I prefer to use this pattern for cases like this.
```suggestion
    const data: ICreatePrismaSession = {
      ...(uid ? { uid } : {}),
      sid,
      expiresAt,
      data: sessionString,
      id: this.dbRecordIdIsSessionId ? sid : this.dbRecordIdFunction(sid),
    };
```

Reply:
I'd been wondering how to handle this more gracefully... Thanks!